src/components/Node.js

```diff
@@ -0,0 +1,44 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { makeGetNodeColor } from '../selectors/protocol';
+import { Node as UINode } from '../ui/components';
+
+const getNodeColor = makeGetNodeColor();
+
+/**
+  * Renders a Node.
+  */
+
+class Node extends PureComponent {
+  render() {
+    const {
+      color,
+    } = this.props;
+
+    return (
+      <UINode
+        color={color}
+        {...this.props}
+      />
+    );
+  }
+}
+
+function mapStateToProps(state, props) {
+  return {
+    color: getNodeColor(state, props),
+  };
+}
+
+Node.propTypes = {
+  /* eslint-disable react/no-unused-prop-types */
+  type: PropTypes.string.isRequired,
+  color: PropTypes.string,
+};
+
+Node.defaultProps = {
+  color: 'node-color-seq-1',
+};
+
+export default connect(mapStateToProps)(Node);
```

Comment:
Since the component only serves to add defaultProps, you could refactor to:

```js
import React, { PureComponent } from 'react';
import { connect } from 'react-redux';
import { compose } from 'recompose';
import PropTypes from 'prop-types';
import { makeGetNodeColor } from '../selectors/protocol';
import { Node } from '../ui/components';

const getNodeColor = makeGetNodeColor();

function mapStateToProps(state, props) {
  return {
    color: getNodeColor(state, props),
  };
}

export default compose(
  defaultProps({
    color: 'node-color-seq-1',
  }),
  connect(mapStateToProps),
)(Node);
```

Reply:
I actually tried the recompose way, but couldn't get it to work. Your code here doesn't work either (it makes `selectable` crap out, along with other errors that I don't really understand. Perhaps we could take a look at this together at a later date?

Reply:
Is it just `defaultProps` that's causing the problem? Sure we can look at that next week.